Resources/Private/Fusion/Prototypes/ActionCollection.Fieldnames.fusion

```diff
@@ -0,0 +1,15 @@
+prototype(Sitegeist.PaperTiger:ActionCollection.Fieldnames) < prototype(Neos.Fusion:Component) {
+    nodePath = null
+    describtion = 'Fieldnames:&nbsp;'
+    renderer = Neos.Fusion:Tag {
+        content = Neos.Fusion:Join {
+            describtion = ${props.describtion}
+            fieldNames = Neos.Fusion:Loop {
+                @glue = ', '
+                items = ${q(node).children(props.nodePath).find('[instanceof Sitegeist.PaperTiger:Field]').get()}
+                itemName = 'field'
+                itemRenderer = ${'{' + q(field).property('name') + '}'}
+            }
+        }
+    }
+}
```

Comment:
This is for the preparation for the clipboard feature: The `button` gets visible when the clipboard feature is available. I will provide also a PR for an CSS file, to make it easier to customize the look and feel in the backend

```suggestion
prototype(Sitegeist.PaperTiger:ActionCollection.Fieldnames) < prototype(Neos.Fusion:Component) {
    nodePath = null
    describtion = 'Fieldnames:&nbsp;'
    renderer = Neos.Fusion:Tag {
        content = Neos.Fusion:Join {
            describtion = ${props.describtion}
            fieldNames = Neos.Fusion:Loop {
                @glue = ', '
                items = ${q(node).children(props.nodePath).find('[instanceof Sitegeist.PaperTiger:Field]').get()}
                itemName = 'field'
                itemRenderer = Neos.Fusion:Component {
                    name = ${'{' + q(field).property('name') + '}'}
                    renderer = afx`
                        <span class="papertiger-clipboard">
                            <button type="button">{props.name}</button>
                            <span>{props.name}</span>
                        </span>
                    `
                }
            }
        }
    }
}
```

Reply:
I ignore this for now as this can/should be done separetely. I kinda dislike the button idea but like the idea of copying the fieldname into the clipboard on click.

This pr aims for a pragmatic solution for now and should not introduce markup the may conflict with styles etc. That is why i choose plaintext here.